app/src/ereferrals/java/org/eyeseetea/malariacare/strategies/SplashActivityStrategy.java

```diff
@@ -56,7 +62,18 @@ public void onClick(DialogInterface dialogInterface, int i) {
     @Override
     public void downloadLanguagesFromServer() throws Exception {
         try {
-            SplashActivityStrategy.super.downloadLanguagesFromServer();
+            if (BuildConfig.downloadLanguagesFromServer) {
+                Log.i(TAG, "Starting to download Languages From Server");
+                CredentialsReader credentialsReader = CredentialsReader.getInstance();
+                IConnectivityManager connectivity = NetworkManagerFactory.getConnectivityManager(
+                        activity);
+                ISettingsRepository settingsRepository = new SettingsDataSource(activity);
+                DownloadLanguageTranslationUseCase downloader =
```

Comment:
Pass settingsRepository as argument to DownloadLanguageTranslationUseCase is not necessary